projects/storefrontlib/src/shared/components/truncate-text-with-popover/truncate.pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'cxTruncate',
+})
+export class TruncatePipe implements PipeTransform {
+  /**
+   * example usage {{ exampleString | truncate: [1, false] }}
+   */
+  transform(value: string, args?: any[]): string {
+    if (!args) {
+      return value;
+    }
+
+    const limit =
+      args.length > 0 &&
+      args[0] !== null &&
+      args[0] !== undefined &&
```

Comment:
we could write `args.length > 0 && ars[0] && Number.isInteger(+args[0])` instead